thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/events/Event.java

```diff
@@ -0,0 +1,54 @@
+package com.linkedin.thirdeye.anomaly.events;
+
+import java.util.List;
+import java.util.Map;
+
+public abstract class Event {
+  String metric;
+  String service;
+
+  long startTime;
+  long endTime;
+
+  Map<String, List<String>> targetDimensionMap;
```

Comment:
I will add javadoc. 
targetDimesionMap will hold metadata of the event. for example 
1. holiday event will have
coutntryCode --> {US, CA, ...}
2. Deployment Event will have
fabric --- > {prod-lva1, prod-ltx1..}
hostname ---> {hosta, hostb, hostc...}